src/main/java/net/cosmocat/marketplace/exception/type/CategoryNotFoundException.java

```diff
@@ -2,15 +2,15 @@
 
 public class CategoryNotFoundException extends ResourceNotFoundException {
 
-  public CategoryNotFoundException(String message) {
-    super(message);
-  }
+    public CategoryNotFoundException(String message) {
+        super(message);
```

Comment:
that's in general bad practice -1, you have here the problem with soc - because the service is building your exceptions, while it should be done by exception itself and you should pass the dynamic part as arg -1